kubemarine/resources/configurations/globals.yaml

```diff
@@ -52,7 +52,7 @@ pods:
       retries: 30
     plugins:
       timeout: 5
-      retries: 30
+      retries: 45
```

Comment:
Probably it's better to increase number of retries more significantly, for example, to 360 (so that the waiting time was up to  a half an hour). This should cover most cases with long starting of pods (calico-node, for example).

Reply:
Done

Reply:
If we expect different time to become ready for different OOB plugins, then we could specify timeout or retries in the defaults.yaml for the `expect` installation procedure directly, for example:
```yaml
        - expect:
            pods:
              list:
                - coredns
                - calico-kube-controllers
                - calico-node
              retries: 360
```

Reply:
Though, even in this case, I'd like to warn that unsuccessful installation will become not friendly, as it will hang the Kubemarine run for half an hour, even if it is technically clear that the pods will never be ready after very smaller time period.

Reply:
Let's have 45 for now. 360 looks too ambisious.